lib/screens/sign_in.dart

```diff
@@ -141,16 +123,14 @@ class _LoginPageState extends State<LoginPage> {
                 height: 80,
               ),
               ElevatedButton(
-                onPressed: _clickForSignIn,
-                style: Theme.of(context).elevatedButtonTheme.style!.copyWith(
-                      backgroundColor: MaterialStateProperty.all<Color>(
-                          CommonColors.lightGreenColor),
-                    ),
+                onPressed: readStore.clickForSignIn,
+                style: theme.elevatedButtonTheme.style!.copyWith(
+                  backgroundColor: MaterialStateProperty.all<Color>(
```

Comment:
If background color is common for the button then put it inside main theme parameter.